### PR TITLE
add lmdb + test

### PR DESCRIPTION
**Pull request by headupinclouds**
Uses lmdb package name.
Package config uses the project() namd liblmdb

https://github.com/ruslo/hunter/issues/1154

- [x] has hunter-packages/lmdb

### CODE REVIEW

**Comment by ruslo** [comment]
> has hunter-packages/lmdb

Done: https://github.com/hunter-packages/lmdb

**Comment by ruslo** [comment]
Testing:
* https://ci.appveyor.com/project/ingenue/hunter/build/1.0.2729
* https://travis-ci.org/ingenue/hunter/builds/300067065?utm_source=github_status&utm_medium=notification

File: cmake/projects/lmdb/hunter.cmake [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+include(hunter_download)
+include(hunter_pick_scheme)
+
+hunter_add_version(
+    PACKAGE_NAME
+    lmdb
+    VERSION
+    0.9.21-p0
+    URL
+    "https://github.com/hunter-packages/lmdb/archive/v0.9.21-p0.tar.gz"
```
Comment (ruslo) [inline cmake/projects/lmdb/hunter.cmake]: I think this release created from the wrong branch:
```
CMake Error: The source directory "/.../9941f34/Build/lmdb/Source" does not
appear to contain CMakeLists.txt.
```

Use this release:
* https://github.com/hunter-packages/lmdb/releases/tag/v0.9.21-p1

Comment (headupinclouds) [reply cmake/projects/lmdb/hunter.cmake]: Done

**Comment by ruslo** [comment]
Testing:
* https://ci.appveyor.com/project/ingenue/hunter/build/1.0.2735
* https://travis-ci.org/ingenue/hunter/builds/300146403?utm_source=github_status&utm_medium=notification

**Comment by headupinclouds** [comment]
* Windows `vs-*` toolchains work, `mingw` fails
* Travis: `analyze` fails

**Comment by ruslo** [comment]
Uploading:
* https://ci.appveyor.com/project/ingenue/hunter/build/1.0.2744
* https://travis-ci.org/ingenue/hunter/builds/300275233?utm_source=github_status&utm_medium=notification

**Comment by ruslo** [comment]
Follow-up: https://github.com/ruslo/hunter/commit/29a3657e1bb0c104f17ff10455bc6a4fb0245770

Released: https://github.com/ruslo/hunter/releases/tag/v0.19.148

**Pull request merged**